sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -1013,3 +1013,7 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
 ('Eye_QP_RETINA_defaults', 'OUVITREOUS_vh', 'vit: hemorrhage', 1020, 0, 0, 'VITREOUS', 'vitreous hemorrhage', '', 0, 0, 0, 'OU');
 #EndIf
+
+#IfNotColumnType layout_options title text
+ALTER TABLE `layout_options` CHANGE `title` `title` TEXT NOT NULL DEFAULT '';
```

Comment:
still fighting the good 5.0.2 fight i see :) 
Couple quick fixes:
1. Make text bare above and in database.sql and in patch.sql (etc. `ALTER TABLE `layout_options` CHANGE `title` `title` TEXT;`)
2. also gotta add to sql/patch.sql

Reply:
actually, i just remembered that you can remove this completely in 5_0_1-to-5_0_2_upgrade.sql and instead just put it in patch.sql (for patches since an upgrade also runs the patch script if it exists).

Reply:
aw man forgot about patch!

Reply:
Out of 8 hours today I think I got to spend only 1 hr on ONC coding!!!!

Reply:
From onc enforcement: (sorry had to find a reason to use this again)

![mp4](https://user-images.githubusercontent.com/5618372/101984312-a64b4b00-3c4e-11eb-996e-293de620d190.gif)